_posts/2024-02-19-jco-1.0.md

```diff
@@ -0,0 +1,290 @@
+---
+title: "Announcing Jco 1.0"
+author: "Yosh Wuyts"
+date: "2024-02-21"
+github_name: "yoshuawuyts"
+excerpt_separator: <!--end_excerpt-->
+---
+
+We’re happy to announce the 1.0 release of [jco]: a native Javascript
+WebAssembly runtime and toolchain built for WebAssembly Components. Jco can
+natively run Wasm Components inside of Node.js, making it easier to take
+libraries written in different programming languages and execute them using the
+Node.js runtime. And by implementing [WASI
```

Comment:
How about something like:

```suggestion
natively run Wasm Components on JavaScript using transpilation, making it easy to take
libraries written in different programming languages and execute them using the
Node.js runtime. And by implementing [WASI
```

Reply:
Right now we only support Node.js; I think it's really important to correctly set expectations from the outset about what it is we're stabilizing. In the next paragraph we cover what is coming next, which includes support for other Javascript runtimes (including the browser).

Reply:
Yeah agreed, thanks for clarifying.